content/en/docs/v3.4/op-guide/configuration.md

```diff
@@ -275,7 +275,7 @@ The security flags help to [build a secure etcd cluster][security].
 + Enable client cert authentication.
 + default: false
 + env variable: ETCD_CLIENT_CERT_AUTH
-+ CN authentication is not supported by gRPC-gateway.
++ CN authentication is not supported by gRPC-gateway. Note if etcd is running as a public service, it's recommended to enable client cert authentication to avoid abuse of http2 for unauthenticated clients.
```

Comment:
Not sure about wording "public service". I think this is more about endpoint availability within a network. Even if the network is trusted, there is a risk that attacker will penetrate it.

```suggestion
+ CN authentication is not supported by gRPC-gateway. Note that running etcd without client cert authentication expose its endpoint to http2 protocol abuse from unauthenticated clients (CVE-2023-44487).
```

Reply:
Running etcd as a public service definitely is more vulnerable the the CVE. 

Updated anyway & thx.